{"metadata":{"externalID":"s:7SwiftPy11PythonErrorO05ValueD0yACSScACmF","modules":[{"name":"SwiftPy"}],"roleHeading":"Case","role":"symbol","title":"PythonError.ValueError(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"ValueError"},{"kind":"text","text":"("},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":")"}],"symbolKind":"case"},"schemaVersion":{"minor":3,"patch":0,"major":0},"variants":[{"paths":["\/documentation\/swiftpy\/pythonerror\/valueerror(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ValueError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":")","kind":"text"}],"languages":["swift"],"platforms":["macOS"]}]}],"hierarchy":{"paths":[["doc:\/\/SwiftPy\/documentation\/SwiftPy","doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError"]]},"kind":"symbol","identifier":{"url":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/ValueError(_:)","interfaceLanguage":"swift"},"references":{"doc://SwiftPy/documentation/SwiftPy":{"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy","title":"SwiftPy","role":"collection","url":"\/documentation\/swiftpy","abstract":[],"kind":"symbol","type":"topic"},"doc://SwiftPy/documentation/SwiftPy/PythonError":{"abstract":[],"role":"symbol","type":"topic","navigatorTitle":[{"text":"PythonError","kind":"identifier"}],"kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"PythonError","kind":"identifier"}],"title":"PythonError","url":"\/documentation\/swiftpy\/pythonerror","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError"},"doc://SwiftPy/documentation/SwiftPy/PythonError/ValueError(_:)":{"kind":"symbol","abstract":[],"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/ValueError(_:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ValueError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}],"role":"symbol","url":"\/documentation\/swiftpy\/pythonerror\/valueerror(_:)","type":"topic","title":"PythonError.ValueError(_:)"}}}